tests/IsoCodes/Tests/CreditCardTest.php

```diff
@@ -30,7 +30,6 @@ public function getValidCreditCards()
             array('4903010000000009'),  //Switch
             array('4111111111111111'),  //Visa
             array('6304100000000008'),  //Laser
-            array(6304100000000008),    //Laser
```

Comment:
This is not removed but added as invalid here: https://github.com/composer/composer/pull/5249#issuecomment-214841671

See https://github.com/ronanguilloux/IsoCodes/issues/75#issuecomment-214868555